### DISCUSSION TITLE
Is a reverse crop possible? That is, add a border?

### DISCUSSION THREAD

**Discussion by ayryq** [aaronwmorris/indi-allsky#1643] [General]
I was thinking it would be nice to move some of the overlay text outside of the image. Is it possible to add a colored or black border on one or more sides of the captured image, in order to have a blank spot to put labels?

**Comment by aaronwmorris** [comment]
I can work on this.

**Comment by aaronwmorris** [comment]
Merged #1665 .  This ended up being a lot more complicated than I originally estimated, but probably not for obvious reasons.

Adding a border is extremely simple, however, several of the functions rely on knowing where the center of the image circle is in the image.  There is already a setting to indicate if the image circle is not the exact center (this is incredibly common), however, if you add a non-uniform border (say the left border is larger than the right border), it makes the calculations more difficult to determine the exact center of the image circle.
